flowtorch/bijectors/utils.py

```diff
@@ -0,0 +1,50 @@
+import functools
```

Comment:
We require a copyright notice at the top of the file (see other source files). There is a CI action that will fail if Python source doesn't contain the copyright at the top...